### DISCUSSION TITLE
When will v12.5 be released?

### DISCUSSION THREAD

**Discussion by paulsuart** [ant-design-blazor/ant-design-blazor#2756] [General]
Hi @ElderJames.  Thanks for approving my PR #2753.  I was wondering when the next release containing this change will be available?

**Comment by ElderJames** [comment]
Next week. But this PR havn't been solved.

**Comment by paulsuart** [comment]
Do the less files still not show up in the nuget package?

I wasn't sure of the process to build the package so was unclear how to test.

**Reply by ElderJames** [reply]
Oh, your PR is not #2753 😂
Anyway, We will release next version in this weekend.

**Comment by paulsuart** [comment]
Oops, sorry!  Thanks for confirming the next release date 😄
